hibernate-core/src/main/java/org/hibernate/Session.java

```diff
@@ -1101,7 +1101,11 @@ public interface Session extends SharedSessionContract, EntityManager {
 	 * @return an instance of {@link IdentifierLoadAccess} for executing the lookup
 	 *
 	 * @throws HibernateException If the given class does not resolve as a mapped entity
+	 *
+	 * @deprecated This method will be removed.
+	 *             Use {@link #find(Class, Object, FindOption...)} instead.
```

Comment:
Seems like we ought to at least link to `FindOption` here.

Or do we just rely on the user "finding implementors" of FindOption?

WDYT?  E.g. something like -

Reply:
```suggestion
	 * @deprecated This method will be removed.
	 *             Use {@link #find(Class, Object, FindOption...)} instead.
	 *             See {@link FindOption}.
```